lib/active_job/queue_adapters/pubsub_adapter.rb

```diff
@@ -7,15 +7,32 @@ class PubsubAdapter
       #
       # @param [ActiveJob::Base] job The job to be performed.
       def enqueue(job)
-        raise(NotImplementedError)
+        puts("\n [PubsubAdapter][enqueue]: #{job.inspect}")
```

Comment:
Could this be solved using instrumentation?

Reply:
Yes, I added the commit https://github.com/skysemilla/kisi-api-challenge/pull/1/commits/8425060cc4cd4472de51ee68f8080f3ed8d6d764 to implement the active support notification to instrument and subscribe to an event.